common/utils/arptable/arp_windows.go

```diff
@@ -8,12 +8,12 @@ package arptable
 // is the same in other Windows versions.
 
 import (
-	"os/exec"
+	"github.com/yaklang/yaklang/common/utils/execx"
 	"strings"
 )
 
 func Table() ArpTable {
-	data, err := exec.Command("arp", "-a").Output()
+	data, err := execx.Command("arp", "-a").Output()
```

Comment:
Creating a new `execx` package is unnecessary; 
This file also include the `go:build windows` comment, simply setting `cmd.SysProcAttr` in this file should suffice.

Reply:
There are several places in the project that use exec, and if possible I would like to find out if other places have the same problem and fix it together.